sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -77,15 +65,47 @@
      */
     private String userNameAttributeName;
 
+    /**
+     * Telemetry data will be collected if true, or disable data collection.
+     */
+    private boolean allowTelemetry = true;
+
+    private Map<String, AuthorizationClientScopesProperties> authorizationClients = new HashMap<>();
+
     /**
      * The all user flows which is created under b2c tenant.
      */
-    private UserFlows userFlows = new UserFlows();
+    private List<String> userFlows = new ArrayList<>();
 
     /**
-     * Telemetry data will be collected if true, or disable data collection.
+     * Specify the primary sign in flow name
      */
-    private boolean allowTelemetry = true;
+    @NotBlank(message = "The primary sign in flow name should not be blank.")
+    private String signInUserFlow;
+
+    @Override
+    public void afterPropertiesSet() {
```

Comment:
if we do the check, there's no need to double-check it in the ` public ClientRegistrationRepository clientRegistrationRepository() {` method